app/src/main/java/org/cuberite/android/fragments/ConsoleFragment.java

```diff
@@ -69,6 +70,7 @@ public void onClick(View v) {
                 }
             }
         });
+        TooltipCompat.setTooltipText(sendCommandButton, getString(R.string.do_execute_line));

```

Comment:
Just a quick uninformed question: Where does this string come from, is it already part of the resources, or did you forget to commit a file?

Reply:
The string is in https://github.com/cuberite/android/blob/master/app/src/main/res/values/strings.xml

Reply:
Ok, so the string was already there. Thx.